CHANGELOG.md

```diff
@@ -53,6 +51,11 @@ request adding CHANGELOG notes for breaking (!) changes and possibly other secti
 ### Deprecations
 
 * The property `polaris.active-roles-provider.type` is deprecated and has no effect anymore.
+- The EclipseLink Persistence implementation has been deprecated since 1.0.0 and will be completely removed
+  in 1.3.0 or in 2.0.0 (whichever happens earlier).
+- The legacy management endpoints at `/metrics` and `/healthcheck` have been deprecated in 1.2.0 and will be
+  completely removed in 1.3.0 or in 2.0.0 (whichever happens earlier). Please use the standard management
```

Comment:
> completely removed in 1.3.0 or in 2.0.0 (whichever happens earlier)

do we wanna give it 1 or 2 more releases for complete removal, 1.5.0 or in 2.0.0 (whichever happens earlier)

I am fine with marking the APIs as forRemoval

Reply:
I would propose to flag to removal now, and the actual removal can happen in Polaris 2.0.0 (I don't see problem to release 2.0.0 soon 😄 ).

Reply:
Postponing removal should be transparent for end users (even if previously said "removing in 1.3 :sweat_smile: ).

Reply:
... and I'm in favour of making 2.0 soon :wink: